README.md

```diff
@@ -39,4 +39,49 @@ ansible-playbook -i inventories/devnet-0/inventory.ini playbook.yaml
-```
\ No newline at end of file
+```
+
+# Spinning up a New Testnet
+To create a new testnet using the infrastructure code, follow these steps:
+
+## Validator Configuration
+1. Open the [main.tf](terraform/devnet-0/main.tf) file located in the terraform/devnet-0/ directory.
+2. Locate the sections that define the different nodes and their corresponding validator ranges, for this example this is `variable "digitalocean_vm_groups"`
+3. Adjust the validator indexes in the terraform/main.tf file based on your desired allocation of validators to nodes.
+
+For example, let's say you want to assign validator index 0-24 to the lodestar-besu-1 node and validator index 25-224 to the lighthouse-nethermind-1 node. Update the main.tf file as follows:
+```terraform
+    {
+      id = "lodestar-besu"
+      vms = {
+        "1" = { ansible_vars : "validator_start=0 validator_end=25" }
+      },
+    },
+    {
+      id = "lighthouse-nethermind"
+      vms = {
+        "1" = { ansible_vars : "validator_start=25 validator_end=225" }
+      },
+    },
+```
+Make sure to adjust the validator ranges according to your requirements and the number of validators in your network. This configuration ensures that validators within the specified ranges will be allocated to the corresponding nodes during the deployment. By customizing the validator indexes in the Terraform configuration, you can allocate validators to specific nodes in your network according to your desired configuration.
+
+4. `terraform apply` will create a the machines and the inventory file for you. The inventory file will be located in the [ansible/inventories/devnet-0](ansible/inventories/devnet-0/) directory.
+5. The inventory.ini file will have the list of all the nodes that were created by Terraform. The inventory file will also have the validator ranges that were specified in the Terraform configuration. The validator ranges will be used by the Ansible playbook to allocate validators to the corresponding nodes.
+```ini
+[lodestar_besu]
+lodestar-besu-1 ansible_host=167.99.34.241 cloud=digitalocean cloud_region=ams3 validator_start=0 validator_end=25
+...
+```
+6. Adjust the total number of validators in the [ansible/inventories/devnet-0/group_vars/all.yaml](ansible/inventories/devnet-0/group_vars/all.yaml) file (ethereum_genesis_generator_config_files.values.env.NUMBER_OF_VALIDATORS) to match with your total number of validators that you are running.. This will be used by the Ansible playbook to generate the validator keys and deposit data for the network.
+
+## Network Configuration
+1. [ansible/inventories/devnet-0/group_vars/all.yaml](ansible/inventories/devnet-0/group_vars/all.yaml) has all the network configuration parameters. Adjust the parameters according to your requirements. Most likely you will not need to adjust these, unless you would like to use a custom setup. The default configuration will work for most networks.
+
+## Deploying the Network
```

Comment:
Could you add a comment at the end of the doc along the lines of "If using Ethpandaops stack" and then specify how the k8s stuff works? Like how do i deploy all the tooling such that argo deploys the stack for me?